force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +107,27 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<string> wordsInSentence = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWordsInSentence = new Set<String>();
 
         // Add each word to the set
+        for(string word : wordsInSentence){
+            uniqueWordsInSentence.add(word);
+        }
 
         // Join the unique words back into a sentence
+        String stringToReturn = '';
+
+        for(String uniqueWord : uniqueWordsInSentence){
+            stringToReturn = stringToReturn + uniqueWord + ' ';
+        }
+        
+        stringToReturn = stringToReturn.trim();
```

Comment:
You can be more efficient and just pass a List directly to the Set constructor and just String. Join and skip both for loops. 

```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
```